docs/source/examples/assignment_workflows/plot_subarea_analysis.py

```diff
@@ -406,3 +407,17 @@ def plot_results(link_loads):
 map = plot_results(rc.get_load_results()["demand"])
 subarea_zone.add_to(map)
 map
+
+# %%
```

Comment:
For these very long examples, it is probably better to add these references right in the beginning.

Changing the top text to: 
Several functions, methods, classes and modules are used in this example: